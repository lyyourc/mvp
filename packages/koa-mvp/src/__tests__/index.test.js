const request = require('supertest')
const Koa = require('../index')

test('koa', async () => {
  const app = new Koa()

  app.use(async (ctx, next) => {
    await next()
    ctx.body.arr.push(2)
  })

  app.use(async (ctx, next) => {
    ctx.body.arr = [1] 
    await next()
  })

  const res = await request(app.listen()).get('/')
  expect(res.body).toMatchSnapshot()
})
