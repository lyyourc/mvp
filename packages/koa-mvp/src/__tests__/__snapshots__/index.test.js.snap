// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`koa 1`] = `
Object {
  "arr": Array [
    1,
    2,
  ],
}
`;
